packages/@ember/-internals/glimmer/tests/integration/helpers/unique-id-test.js

```diff
@@ -1,8 +1,34 @@
-import { RenderingTestCase, strip, moduleFor, runTask } from 'internal-test-helpers';
+import {
+  AbstractTestCase,
+  RenderingTestCase,
+  strip,
+  moduleFor,
+  runTask,
+} from 'internal-test-helpers';
 import { setProperties } from '@ember/object';
+import { uniqueId } from '@ember/helper';
 import { EMBER_UNIQUE_ID_HELPER } from '@ember/canary-features';
 
 if (EMBER_UNIQUE_ID_HELPER) {
+  moduleFor(
+    'Helpers test: {{unique-id}} JS',
+    class JSRenderingTest extends AbstractTestCase {
+      constructor(assert) {
+        super(assert);
+        this.assert = assert;
+      }
+      ['@test it can be invoked as a JS function']() {
+        let first = uniqueId();
+        let second = uniqueId();
```

Comment:
Here, then, you should be able to use those new imports like this:

```suggestion
        let first = getValue(invokeHelper({}, uniqueId()));
        let second = getValue(invokeHelper({}, uniqueId()));
```

The rest of the test *should* “just work” once you’ve added that in. This gets at the bit we’ll need to do follow-up work around—this obviously isn’t a great experience in JS! However, we don’t need to tackle that here, and it needs some broader design work!

Reply:
Hrm, assuming we want just `uniqueId` since its not fn. So trying 
`getValue(invokeHelper({}, uniqueId)` I get the following very intentional sounding error.

```
Died on test #1: Found a helper manager, but it was an internal built-in helper manager. `invokeHelper` does not support internal helpers yet.
```

I dug into `@glimmer/runtime` a bit, and it seems like we just determine this to be an internal built-in helper manager based on the fact that it takes in a type function.

Here's the minimal repro of the situation:

```        
   let myRef = internalHelper(() => createConstRef('foo', 'unique-id'));
    try {
      invokeHelper({}, myRef);
    } catch (ex) {
      console.error('see error:', ex);
    }
```

I am missing some context of what an internal helper is vs external(?). Had this just been a `Reference`, I see there are helper functions like `valueForRef(createConst('foo', 'unique-id')) // => foo`